CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 ## Unreleased
+- Add `InvalidAuthentication` to `ExceptionThrower`
```

Comment:
It should have a `### Added` section under `## Unreleased`` and the changes should added there.

Guideline: https://keepachangelog.com/en/1.1.0/

Reply:
good catch, fixed